apps/bare-expo/android/keystores/debug.keystore.properties

```diff
@@ -0,0 +1,4 @@
+key.store=debug.keystore
```

Comment:
For Google Maps and other Google APIs, we need to use a shared keystore during development. We already have a debug keystore that we use for the Expo client -- whether it's through this properties file or the Gradle file, make the debug build use our shared keystore.

Reply:
ELI5